cumulusci/robotframework/CumulusCI.py

```diff
@@ -47,12 +49,17 @@ def __init__(self, org_name=None):
             logging.WARN
         )
 
+    @property
```

Comment:
I like that you added this property 👍 

Though, personally I'd like to see an explicit `return None` just so that it's clear that's what you intend to happen and not that you forgot to handle that case.

Reply:
I literally had the return None in my code for a week and then deleted it just before the checkin because I guessed that someone was going to complain that it was redundant and unnecessarily verbose...

That's the problem with trying to mind-read!

Reply:
https://github.com/SFDO-Tooling/CumulusCI/commit/7d0c5e80800f53ac79ea9b4d4f76d12406df123f

Reply:
> I literally had the return None in my code for a week and then deleted it just before the checkin because I guessed that someone was going to complain that it was redundant and unnecessarily verbose...

Hah! I do that too sometimes. I write code, I'm happy with it, I use it, and then at the last second before a commit I think "ahh, I should change this".